profile-widget/divepixmapitem.cpp

```diff
@@ -25,6 +25,7 @@ CloseButtonItem::CloseButtonItem(QGraphicsItem *parent): DivePixmapItem(parent)
 
 void CloseButtonItem::mousePressEvent(QGraphicsSceneMouseEvent *event)
 {
+	Q_UNUSED(event);
```

Comment:
In this case perhaps it would be less noisy to simply omit the parameter name as in:
```
void CloseButtonItem::mousePressEvent(QGraphicsSceneMouseEvent *)
```
@dirkhh: Is there a guideline concerning unused parameters? I see at least 3 versions in the codebase:
1) Q_UNUSED(param)
2) (void)param;
3) Omission of name

Since we already depend on 2) and 3), is there a point in using Qt's supposedly platform-independent version?

Reply:
`(void)param` is the way of doing it outside of Qt - e.g. in C.
but also ... `#define Q_UNUSED(x) (void)x;`:
https://github.com/qt/qtbase/blob/5.11/src/corelib/global/qglobal.h#L117

both `Q_UNUSED` and omitting are fine here, omitting is better as it saves a line.

Reply:
@dirkhh is we decide for one standard e.g. 3. where possible else 2., I volunteer to do code cleanup, since it gives me a reason to visit other code parts.